tests/acceptance/30_generic_methods/unsafe/service_started.audit.cf

```diff
@@ -0,0 +1,74 @@
+#######################################################
+#
+# Test if cron is started and if no, start it
+#
+#######################################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("${this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+    debian|suse::
+      "service_name"  string => "cron";
+    redhat::
+      "service_name"  string => "crond";
+    aix::
+      "service_name"  string => "syslogd";
+    any::
+      "unknown_service_name" string => "unknown_service_name";
+
+  methods:
+      "serverspec" usebundle => execute_serverspec("${init.service_name}_started.rb", "service_already_started", "0");
```

Comment:
numbering for the comment is misleading - i kind of believed it was for exit code